lib/ukrust/ukrust_sys/allocator.rs

```diff
@@ -79,3 +79,6 @@ pub fn __rust_alloc_zeroed(size: usize, _align: usize) -> *mut u8 {
 pub fn __rust_alloc_error_handler(_size: usize, _align: usize) -> ! {
     panic!("Alloc error handler");
 }
+
+#[no_mangle]
+static __rust_alloc_error_handler_should_panic: u8 = 1;
```

Comment:
This should not be necessary. This is consistent with the other `__rust_` symbols, though, so removing them can be a separate effort in my opinion.

Reply:
Ah, I see. Together with the other comment, this can then be tackled in another PR. This matches the current approach, but can be circumvented, by using a more suitable target.